src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://a.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+A user visits a page (`https://a.example`) which requests an image
+(`https://x.example/doge.png`). In this case, the image is requested from the
+network and cached using `https://a.example` (the top-level site) +
+`https://a.example` (the current-frame site) + `https://x.example/doge.png` (the
+resource URL) as the key. (Note that when the resource request is from the
+top-level -frame, the top-level site and current-frame site in the Network
+Isolation Key are the same.)
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-2.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://b.example</code>, <code>https://b.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+The same user visits a different page (`https://b.example`) which requests the
+same image (`https://x.example/doge.png`). Though the same image was loaded in
+the previous example, since the key doesn't match it will not be a cache hit.
+
+The image is requested from the network and cached using `https://b.example` +
+`https://b.example` + `https://x.example/doge.png` as the key.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-6.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://a.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+Now the user comes back to `https://a.example` but this time the image
+(`https://x.example/doge.png`) is embedded within an iframe. In this case, the
+key is `https://a.example` + `https://a.example` + `https://x.example/doge.png`
+and a cache hit occurs. (Note that when the top-level site and the iframe are
+the same site, the resource cached with the top-level frame can be used.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-4.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+The user is back at `https://a.example` but this time the image is hosted in an
+iframe from `https://c.example`.
+
+In this case, the image is downloaded from the network because there is no
+resource in the cache that matches the key `https://a.example` +
+`https://c.example` + `https://x.example/doge.png`.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-7.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
```

Comment:
I think you should come up with a single way to represent keys and use it consistently.

Reply:
I don't understand your suggestion. I think it's already consistent?

Reply:
In the previous paragraph you use + signs instead of showing it as a tuple.

Reply:
Again, please have a look at how this will look on the website: https://pr-9038-dot-web-central.appspot.com/web/updates/2020/10/http-cache-partitioning
I'm consistent on the tuples below the image while + on the paragraph to the right.